pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '9.0'
```

Comment:
Could you update pnpm to version 9.0, and then re-run `pnpm i` to regenerate the lockfile?